src/components/panels/scenes.vue

```diff
@@ -6,6 +6,7 @@
 
 		<button
 			v-for="scene in scenes"
+			v-if="!scene.name.startsWith(hidePrefix)"
```

Comment:
[v-for + v-if isn't really recommended in vue styleguide](https://vuejs.org/v2/style-guide/#Avoid-v-if-with-v-for-essential), put this into computed property like `filteredScenes`?